challenges/objects-arrays.js

```diff
@@ -6,25 +6,50 @@
   object name, diet, weight, length, period
 */
 
-// tyrannosaurus, carnivorous, 7000kg, 12m, Late Cretaceious
+const makeDino = function (obj){
 
-// stegosaurus, herbivorous, 2000kg, 9m, Late Jurassic
+  this.name = obj.name ;
+  this.diet = obj.diet;
+  this.weight = obj.weight;
+  this.length = obj.length ;
+  this.period = obj.period;
 
-// velociraptor, carnivorous, 15kg, 1.8m, Late Cretaceious
+}
 
+
+
+// tyrannosaurus, carnivorous, 7000kg, 12m, Late Cretaceious
+const tRex = new makeDino({name: 'Tyrannosaurus', diet: 'carnivorous', weight: '700kg', length: '12m', period: 'Late Cretaceous',})
+// stegosaurus, herbivorous, 2000kg, 9m, Late Jurassic
+const Stego = new makeDino({name: 'Stegosaurus', diet: 'herbivorous', weight: '2000kg', length: '9m', period: 'Late Jurassic', })
+// velociraptor, carnivorous, 15kg, 1.8m, Late Cretaceious
+const vRaptor = new makeDino({name: 'Velociraptor', diet: 'carnivorous', weight: '15kg', length: '1.8m', period: 'Late Cretaceous',})
 // Using your dinosaur objects, log answers to these questions:
 
 // How much did tyrannosaurus weigh?
-console.log(tyrannosaurus.weight);
+console.log(tRex.weight);
 // What was the diet of a velociraptor?
-console.log(velociraptor.diet);
+console.log(vRaptor.diet);
 // How long was a stegosaurus?
-console.log(stegosaurus.length);
+console.log(Stego.length);
 // What time period did tyrannosaurus live in?
-console.log(tyrannosaurus.period);
+console.log(tRex.period);
+
 
 // Create a new roar method for the tyrannosaurus.  When called, return "RAWERSRARARWERSARARARRRR!" Log the result.
-console.log(tyrannosaurus.roar());
+
```

Comment:
Overall, I like that you went the extra mile and basically created a prototype of Dinosaurs - the prompt basically just asked to create three objects, which you did here just fine...this this block was not necessarily needed, but it may have saved a few keystrokes.